security/oc_pstat.c

```diff
@@ -117,9 +117,6 @@ valid_transition(size_t device, oc_dostype_t state)
       return false;
     break;
   case OC_DOS_RFPRO:
-    if (state == OC_DOS_RFOTM)
-      return false;
-    break;
```

Comment:
why this is removed ? this code seems ok

Reply:
It's exactly the same as the next option (`OC_DOS_RFNOP`), so clang-tidy complained about a duplicated case in the switch.